### PR TITLE
Add readme for AnnData downloads

### LINKED ISSUE #456: Add AnnData template readme for single-cell downloads
### Context

We will now be offering the option to download either data as `SingleCellExperiment` or as `AnnData`. The files included in those downloads will be slightly different and right now the `README` is tailored to `SingleCellExperiment`. 

### Problem or idea

We need to create a new `README` that will be included with `AnnData` downloads. 

### Solution or next step

Create an additional template `README` file that describes the contents of the single-cell downloads if `AnnData` is chosen as the format.

### PR DESCRIPTION
**Pull request by allyhawkins**
## Issue Number
Closes #456 

## Purpose/Implementation Notes

This PR adds in a new template readme for AnnData downloads, `readme_anndata.md`. 
Mostly I mirrored the content of what's already included in the main `readme.md` for single-cell downloads. 
The only major modifications I made was including a description of the CITE-seq files since those are separate files for AnnData, but not for SCE objects. 

I also included the FAQ on working with the files in Python. Right now, that link won't work because the changes are in `development`. But it will work once those changes get merged in. 
And then the last thing I did was add in a link for the getting started docs. 

## Types of changes

What types of changes does your code introduce?

- New feature (non-breaking change which adds functionality)

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by arkid15r — COMMENTED** [review]
LGTM

// I guess you're more interested in Josh's review here.

File: api/scpca_portal/templates/readme_anndata.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Most samples only have one library that has been sequenced.
+See the [FAQ section about samples and libraries](https://scpca.readthedocs.io/en/latest/faq.html#what-is-the-difference-between-samples-and-libraries) for more information.
+
+The files associated with each library are (example shown for a library with ID `SCPCL000000`):
+- An unfiltered counts file: `SCPCL000000_unfiltered_rna.hdf5`,
+- A filtered counts file: `SCPCL000000_filtered_rna.hdf5`,
+- A processed counts file: `SCPCL000000_processed_rna.hdf5`,
+- A quality control report: `SCPCL000000_qc.html`
+
+Also included in each download is a `single_cell_metadata.tsv`, a tab-separated table, with one row per library and columns containing pertinent metadata corresponding to that library.
```
Comment (jashapiro) [inline api/scpca_portal/templates/readme_anndata.md:19]:
```suggestion
Also included in each download is `single_cell_metadata.tsv`, a tab-separated table, with one row per library and columns containing pertinent metadata corresponding to that library.
```

File: api/scpca_portal/templates/readme_anndata.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+For instructions on using the HDF5 files, please see [FAQ: How do I use the provided HDF5 files in Python?](https://scpca.readthedocs.io/en/latest/faq.html#how-do-i-use-the-provided-HDF5-files-in-python)
+For more information on working with the processed `AnnData` objects, see [`Getting started with an ScPCA dataset`](https://scpca.readthedocs.io/en/latest/getting_started.html).
+
+## Contact
+
+If you identify issues with this download, please [file an issue on GitHub.](https://github.com/AlexsLemonade/scpca-portal/issues/new) If you would prefer to report issues via e-mail, please contact us at [scpca@ccdatalab.org](mailto:scpca@ccdatalab.org).
+
+## Citing
+
+To cite data from {project_accession}, please see the citation information at [{project_accession} page.]({project_url})
```
Comment (jashapiro) [inline api/scpca_portal/templates/readme_anndata.md:43]: We seem to mostly put the periods outside the links, so I'm doing that here (other readme may need updating too?)
```suggestion
To cite data from {project_accession}, please see the citation information at [{project_accession} page]({project_url}).
```

File: api/scpca_portal/templates/readme_anndata.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Terms of Use
+
+In using these data, you agree to our [Terms of Use.](https://scpca.alexslemonade.org/terms-of-use)
```
Comment (jashapiro) [inline api/scpca_portal/templates/readme_anndata.md:47]:
```suggestion
In using these data, you agree to our [Terms of Use](https://scpca.alexslemonade.org/terms-of-use).
```

**Review by jashapiro — APPROVED** [review]
LGTM, with just a couple tiny things that might need updates in the other readme file as well if you copied from there (as I hope you did!).

**Pull request merged**